docs/examples/custom_task.ts

```diff
@@ -1,4 +1,4 @@
-import tf from '@tensorflow/tfjs'
+import tf from '@tensorflow/tfjs-node'
```

Comment:
This is probably due to something else than this import but running `npm run custom task` hangs forever (or at least a long time)

Reply:
hum, it's already the case in develop, is it not wanted? from a quick look, it seems to be only starting a server and never closing it (node doesn't exit as long a their are some stuff/promises running)